### ISSUE TITLE
Report: Count of live animals in the program that are larger than 25mm

### ISSUE THREAD

**Issue by CraigJZ** [rubyforgood/abalone#788]
Create queries that return a count of live animals in the program that are larger than 25mm by cohort, by facility and by organization.

Data for this is collected on a subset on animals and would need to be extrapolated to a full group. Some discussion with the stakeholder may be needed to ensure this is done accurately.

Depends on #715 (making mortality available to Blazer queries)

**Comment by CraigJZ** [comment]
Thanks @web-kat!!

**Comment by CraigJZ** [comment]
Sorry - I accidentally closed this when I merged the associated PR.

Yes! This looks like it will resolve the issue. The flexibility of it is really helpful too. :smile:

I don't think we need to check the animals.organization_id on line 29 because the row level security _should_ prevent returning data from other organizations. Having it there won't hurt anything, though, so I'd be cool with leaving it as is.

I've added the new seed data to the staging server, so this one should be ready to add there. Thank you for all the work on this issue!
